diagnostics/matlab/plot_ens_err_spread.m

```diff
@@ -71,7 +71,7 @@ function plot_ens_err_spread(input_file)
 switch lower(pinfo.model)
 
    case {'9var','lorenz_63','lorenz_84','lorenz_96','lorenz_96_2scale', ...
-	 'forced_lorenz_96','lorenz_04','ikeda','simple_advection', 'null'}
+	 'forced_lorenz_96','lorenz_04','ikeda','simple_advection','lorenz_96_tracer_advection', 'null'}
```

Comment:
this is a comment for future redesign of the matlab diagnostics.  there's something not right about how this was implemented if adding a new model to the diagnostics requires every .m file to have the case statement updated with the name of the model.  

for example, the pinfo structure could define categories of models (e.g. 1d, 3d, etc), in addition to the model name.  to add a new model that is similar to an existing one, you set the appropriate category string.  then these files have a case statement on pinfo.category.  as long as you're not adding a model that needs different code, you wouldn't have to change the .m files at all.